OracleTuxedo/core/samples/ws_ssl_svr/simpcl.c

```diff
@@ -0,0 +1,85 @@
+/*
+** Copyright (c) 2022 Oracle and/or its affiliates.
+** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+*/
+
+/*	Copyright (c) 1997 BEA Systems, Inc.
+  	All rights reserved
+
+  	THIS IS UNPUBLISHED PROPRIETARY
+  	SOURCE CODE OF BEA Systems, Inc.
+  	The copyright notice above does not
+  	evidence any actual or intended
+  	publication of such source code.
+*/
```

Comment:
This text suggests that the code is not actually open source and cannot be published, which contradicts the other header above it. If this code has been open sourced, please remove this block, though you've already published it publicly.